octoprint_mrbeam/filemanager/__init__.py

```diff
@@ -33,7 +38,7 @@ def save(self, absolute_dest_path):
 
     def __init__(self, plugin):
         self._plugin = plugin
-        self._logger = plugin._logger
+        self._logger_mrb = mrb_logger("octoprint.plugins.mrbeam.filemanager")
```

Comment:
Why change from the standard `self._logger` ?

Reply:
this class is inhertited from OP. Either my mrb_logger is overwritten by OP or OPs is overwritten with mine. Intentionally used two different ones.